packages/icon/package.py

```diff
@@ -0,0 +1,247 @@
+    German Weather Service and the
+    Max Planck Institute for Meteorology for
+    developing a unified next-generation global numerical weather prediction and
+    climate modelling system. The ICON model has been introduced into DWD's
+    operational forecast system in January 2015."""
+
+    homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
+    url = 'https://gitlab.dkrz.de/icon/icon-cscs/-/archive/mc10_for_icon-2.6.x-rc/icon-cscs-mc10_for_icon-2.6.x-rc.tar.gz'
+    git = 'git@gitlab.dkrz.de:icon/icon-cscs.git'
+
+    maintainers = ['egermann']
+
+    version('master', branch='master', submodules=True)
+    version('ham', git='git@git.iac.ethz.ch:hammoz/icon-hammoz.git', branch='hammoz/gpu/master', submodules=True)
+    version('2.6.x-rc', commit='040de650', submodules=True)
+    version('2.0.17', commit='39ed04ad', submodules=True)
+
+    depends_on('m4')
+    depends_on('autoconf%gcc')
+    depends_on('automake%gcc')
+    depends_on('libtool%gcc')
+    depends_on('cmake%gcc')
+    depends_on('libxml2@2.9.7%gcc', type=('build', 'link', 'run'))
+    depends_on('serialbox@2.6.0', when='+serialize', type=('build', 'link', 'run'))
+    depends_on('eccodes@2.18.0 +build_shared_libs', when='+eccodes', type=('build', 'link', 'run'))
+    depends_on('hdf5', type=('build', 'link', 'run'))
+    depends_on('claw@2.0.1', when='+claw', type=('build', 'link', 'run'))
+    depends_on('netcdf-c ~mpi', type=('build', 'link', 'run'))
+    depends_on('mpicuda', when='icon_target=gpu', type=('build', 'link', 'run'))
+    depends_on('ompt-openmp', when='%pgi', type=('build', 'link', 'run'))
+    depends_on('mpi', when='icon_target=cpu', type=('build', 'link', 'run'))
+    depends_on('cuda%gcc', when='icon_target=gpu', type=('build', 'link', 'run'))
+
+    variant('icon_target', default='gpu', description='Build with target gpu or cpu', values=('gpu', 'cpu'), multi=False)
+    variant('host', default='daint', description='Build on host daint', multi=False)
+    variant('cuda_arch', default='60', description='Build with cuda_arch', values=('70', '60', '37'), multi=False)
+    variant('claw', default=True, description='Build with claw directories enabled')
+    variant('rte-rrtmgp', default=True, description='Build with rte-rrtmgp enabled')
+    variant('mpi-checks', default=False, description='Build with mpi-check enabled')
+    variant('openmp', default=True, description='Build with openmp enabled')
+    variant('serialize', default=False, description='Build with serialization enabled')
+    variant('eccodes', default=False, description='Build with grib2 enabled')
+    variant('test_name', default='none', description='Launch test: test_name after installation')
+    variant('skip-config', default=False, description='Skip configure phase')
+    variant('ham', default=True, description='Build with hammoz and atm_phy_echam enabled.')
+
+    conflicts('+openmp', when='%intel')
+    conflicts('+openmp', when='%cce')
```

Comment:
I know there are problems with CCE in general, but why is openmp and cce not allowed? CCE used to be the reference for performance benchmarking because it produced faster code for CPU than PGI did.